include/multipass/constants.h

```diff
@@ -71,6 +71,11 @@ constexpr auto petenv_default = "primary";
 constexpr auto timeout_exit_code = 5;
 
 constexpr auto authenticated_certs_dir = "authenticated-certs";
+
+constexpr auto driver_deprecation_warning_template =
+    "**Warning! The {0} driver is deprecated and will be removed in an future release.**\n\n"
+    "After the removal, you will find the guide on how to access the {0} instances on \n"
+    "https://documentation.ubuntu.com/multipass/en/latest/\n\n"; // TODO lxd and libvirt migration, remove
```

Comment:
I like that we point to (future) documentation. However, maybe we could tune this to the driver and say:

- for libvirt: "Your instances will remain available with the QEMU driver"
- for LXD: "Your instances will no longer be available in Multipass then, but they will remain in LXD.

Then, when we actually remove the backend, we can say "... driver has been removed. Multipass is now using QEMU. ":

- for libvirt: "Your instances remain available."
- for LXD: "Your instances are still available in LXD. To access them, use `lxc --project=multipass ...`. `lxc --help` for more.